.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "faiss-sys/faiss"]
+	path = faiss-sys/faiss
+	url = https://github.com/Enet4/faiss.git
```

The submodule should point to branch `c_api_head`.
```suggestion
	url = https://github.com/Enet4/faiss.git
	branch = c_api_head
```